src/tools/CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ set(rados_srcs
   ${PROJECT_SOURCE_DIR}/src/common/obj_bencher.cc
   ${PROJECT_SOURCE_DIR}/src/osd/ECUtil.cc)
 add_executable(rados ${rados_srcs})
+if(WITH_LIBRADOSSTRIPER)
 target_link_libraries(rados librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS} radosstriper)
+else(WITH_LIBRADOSSTRIPER)
+target_link_libraries(rados librados global cls_lock_client ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS})
```

Comment:
please avoid repeating. and why shall we add `cls_lock_client` here? also wrong indent. better off:
```cmake
target_link_libraries(rados librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS})
if(WITH_LIBRADOSSTRIPER)
  target_link_libraries(rados radosstriper)
endif()
```

Reply:
1) I'm not seeing what's repeated? (I also haven't had coffee yet, I'll keep looking.)
2) cls_lock_client is normally linked in via libradosstriper. Without libradosstriper, it has to itself be linked in. That said, I'm not really familiar with what that library's purpose is: if it makes no sense /without/ libradosstriper, then I'll make its usage conditional and this goes away. 
3) I followed the indentation used in the rest of the CMakeList.txt file, but I think it's easier to read the way you suggest. Shall I change everything to this indent style?

Reply:
@tchaikov Looks like it's just the indentation of the examples near my patch-- will change.

Reply:
`cls_lock_client` is already linked into librados, see https://github.com/ceph/ceph/blob/master/src/librados/CMakeLists.txt#L31.

Reply:
There was some discussion on the ceph-devel list about this:
https://marc.info/?t=153263095400001&r=1&w=4

Have you tried compiling with -DWITH_LIBRADOSSTRIPER=0? Without cls_lock_client, it fails to link for me.